test/ejabberd_tests/tests/muc_light_legacy_SUITE.erl

```diff
@@ -64,6 +64,7 @@
 -type ct_aff_users() :: [ct_aff_user()].
 -type ct_block_item() :: {What :: atom(), Action :: atom(), Who :: binary()}.
 -type verify_fun() :: muc_helper:verify_fun().
+-type xmlel() :: #xmlel{}.
```

Comment:
There is an exported type for this already: `exml:element()`. I agree juxt `xmlel()` is shorter. Let's make the internal `xmlel()` type an alias for `exml:element()`